ra_aid/server/api_v1_spawn_agent.py

```
@@ -2,6 +2,7 @@
 
 import threading
 import logging
+import json # Added for step_data serialization
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)